web/themes/new_weather_theme/assets/js/charts/hourly-temperature.js

```diff
@@ -0,0 +1,125 @@
+/* global Chart ChartDataLabels */
+(() => {
+  const fontMono = `DM Mono, "Roboto Mono Web", "Courier New", monospace, serif`;
+
+  const colors = {
+    base: "#71767A",
+    baseLighter: "#DFE1E2",
+    baseLightest: "#F5F6F7",
+    primary: "#005EA2",
+    primaryDark: "#0B4778",
+    primaryLight: "#0085CA",
+  };
```

Comment:
@eric-gade Is this where we could import CSS vars? I'd love to not redefine things!

Reply:
Ok, so here is what we would need to do (according to my slapdash thinking):
1. Update our scss file so we map the SASS variables we care about to CSS custom properties (aka CSS variables). It would look something like:
```scss
canvas.whatever-class-for-visual-charts {
  --base-color: #{$base}; // or whatever the color variable is called
  --base-lighter-color: #{$baseLighter};
  // etc etc
}
```
Then in the code where we load the canvas, we do something like:
```javascript
const getColors = () => {
  const canvasEl = getCanvasElement(); // fake thing, get the element
  const computed = getComputedStyle(canvasEl);
  return {
    base: computed.getPropertyValue("--base-color"),
    baseLighter: computed.getPropertyValue("--base-lighter-color"),
    // etc etc
  }
}
```

Reply:
Oops didn't read the full file in the code you linked to. None of this has to be on the canvas element, it can be on any element we control, including whatever containers we create for the charts

Reply:
Ooh, that's cool. I'm gonna go back in and fiddle with that right now. Thanks!